chaoslib/litmus/platform/gke/node_cpu_consumption.yml

```diff
@@ -43,6 +47,8 @@
       template:
         src:  /chaoslib/litmus/platform/gke/node-cpu-hog-job.j2
         dest: /chaoslib/litmus/platform/gke/node-cpu-hog-job.yml
+      vars:
+        cpu_limit: "{{ node_cpu_core }}"
```

Comment:
Here it will pass `node_cpu_core` as `cpu_ltmit` always, Independent of whether `node_cpu_core` is defined or not. so it will fail if you will not provide `node_cpu_core`. can we resolve it?

Reply:
[when_node_cpu_core_is_not_defined.txt](https://github.com/litmuschaos/litmus/files/4363202/when_node_cpu_core_is_not_defined.txt)

Reply:
It seems like if I pass an undefined variable it takes the last assigned value only which is our need.

Reply:
Lets make it explicit via `when`?